src/cs/LionWeb.Integration.WebSocket.Tests/WebSocketClientTestBase.cs

```diff
@@ -74,29 +101,35 @@ private void StartServer()
             Thread.Sleep(100);
         }
 
-        Assert.IsFalse(_process.HasExited);
+        ClassicAssert.IsFalse(_process.HasExited);
     }
 
     protected virtual string AdditionalServerParameters() =>
         "";
 
-    /// <inheritdoc />
-    [TestCleanup()]
-    [ClassCleanup(InheritanceBehavior.BeforeEachDerivedClass, ClassCleanupBehavior.EndOfClass)]
-    public void Dispose()
+    private static readonly List<Process> _allServers = []; 
+    
+    [TearDown]
+    [OneTimeTearDown]
+    public void StopServer()
```

Comment:
`StopServers` (plural)?

Reply:
In principle we only want to stop one server (as these are highlander-servers). The edge case of several servers only happens if we execute several tests together, and for some reason the previous one has not been killed.